tox.ini

```diff
@@ -12,26 +12,27 @@ allowlist_externals =
     echo
 setenv =
     MPLBACKEND = agg
-    COLUMNS = 180
+    COLUMNS = 80
     PIP_INDEX_URL = {env:PIP_INDEX_URL:https://pypi.anaconda.org/scipy-wheels-nightly/simple}
     PIP_EXTRA_INDEX_URL = {env:PIP_EXTRA_INDEX_URL:https://pypi.org/simple}
-    PYTEST_COMMAND = pytest --pyargs plasmapy --durations=5 -n=auto --dist=loadfile --tb=short
-    BUILD_REQS = python -m uv pip compile pyproject.toml --upgrade
+    PYTEST_COMMAND = pytest --pyargs plasmapy --durations=5 --tb=short
+
 extras = tests
 deps =
     astropydev: git+https://github.com/astropy/astropy
     matplotlibdev: git+https://github.com/matplotlib/matplotlib
     numpydev: numpy
     sphinxdev: git+https://github.com/sphinx-doc/sphinx
     xarraydev: git+https://github.com/pydata/xarray
-    cov: pytest-cov
-    !minimal: pytest-xdist
-    pytest-github-actions-annotate-failures
+    pytest_cov_all: pytest-cov
+    # pytest-github-actions-annotate-failures
 commands =
-    !all: {env:PYTEST_COMMAND} {posargs} -m 'not slow'
-    all: {env:PYTEST_COMMAND} {posargs}
-    cov-all: {env:PYTEST_COMMAND} {posargs} --cov=plasmapy --cov-report=xml --cov-config={toxinidir}{/}pyproject.toml --cov-append --cov-report xml:coverage.xml
-    doctest: {env:PYTEST_COMMAND} {posargs} --doctest-modules --doctest-continue-on-failure
+    pytest_all: {env:PYTEST_COMMAND} {posargs}
+    pytest_skipslow: {env:PYTEST_COMMAND} {posargs} -m "not slow"
+    pytest_doctests_all: {env:PYTEST_COMMAND} {posargs} --doctest-modules --doctest-continue-on-failure
+    pytest_doctests_skipslow: {env:PYTEST_COMMAND} {posargs} --doctest-modules --doctest-continue-on-failure -m "not slow"
+    pytest_cov_all: {env:PYTEST_COMMAND} {posargs} --cov=plasmapy --cov-report=xml --cov-config={toxinidir}{/}pyproject.toml --cov-append --cov-report xml:coverage.xml
```

Comment:
A problem with the previous setup was that having both `!all` and `all` made it so that pytest ended up being run twice in some environments.

I wasn't able to find a good explanation in the tox documentation about how the `command` field depended on the names of environments for the generative whatnot, so I went with having unique strings and including pytest in it.

This is making me want to switch to `nox`...